src/plugins/explore/public/components/query_panel/query_panel_widgets/query_panel_widgets.tsx

```diff
@@ -32,14 +24,6 @@ export const QueryPanelWidgets = () => {
         {/* TODO: Actions should go here */}
         <QueryPanelError />
       </div>
-
-      {/* Right Section */}
-      <div className="exploreQueryPanelWidgets__right">
-        <SelectedLanguage />
-        <LanguageToggle />
```

Comment:
why are we removing `SelectedLanguagte` an `LanguageToggle`?

Reply:
Good catch. I think it got removed due to rebase. Let me add them back.